librosa/core/spectrum.py

```diff
@@ -469,7 +469,7 @@ def ifgram(y, sr=22050, n_fft=2048, hop_length=None, win_length=None,
     # Window for discrete differentiation
```

Comment:
maybe comment on the importance of cyclic differentiation? or even provide a ref to #888